addons/hr_expense/static/src/views/kanban.xml

```diff
@@ -18,6 +18,10 @@
     </t>
 
     <t t-name="hr_expense.KanbanButtons" t-inherit="web.KanbanView.Buttons" t-inherit-mode="primary">
+        <xpath expr="//div[hasclass('o_cp_buttons')]" position="attributes">
+            <attribute name="class" remove="d-flex" separator=" "/>
+            <attribute name="class" add="d-block d-xl-flex" separator=" "/>
+        </xpath>
```

Comment:
@oco-odoo This is the only real change (other than the one we discussed) made, to avoid the current behavior in the dropdown and instead display the buttons vertically
![2024-07-16_14-55](https://github.com/user-attachments/assets/f05464a0-19cc-4be0-80c7-4fe81030132e)